lib/tasks/export.rake

```diff
@@ -0,0 +1,15 @@
+require 'csv'
+
+namespace :host_reports do
+  task :export => :environment do
+    CSV.open("reports.csv", "w") do |csv|
+      ConfigReport.all.each do |i|
+        next if i.type != "ConfigReport"
+
+        csv << [i.id, i.host_id, i.reported_at, i.status.to_json,
```

Comment:
This is the model:

=> ConfigReport(id: integer, host_id: integer, reported_at: datetime, created_at: datetime, updated_at: datetime, status: integer, metrics: text, type: string, openscap_proxy_id: integer, origin: string)